docs/docsite/rst/reference_appendices/automationhub.rst

```diff
@@ -5,9 +5,7 @@ Ansible Automation Hub
 
 .. important::
```

Comment:
This line also has to go, and the indent for the next line also needs to be removed.